codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/protocols/serialize/QuerySerializerGenerator.kt

```diff
@@ -117,8 +117,6 @@ abstract class QuerySerializerGenerator(private val codegenContext: CodegenConte
     }
 
     override fun payloadSerializer(member: MemberShape): RuntimeType {
-        // TODO(EventStream): Query payload serialization is required for RPC initial message as well as for message
-        // frames that have a struct/union type.
         TODO("$protocolName doesn't support payload serialization yet")
```

Comment:
Is there an existing ticket we could reference here? If not, we can leave this as is.

Reply:
Nope, there isn't.